pkg/appmanager/appManager_test.go

```diff
@@ -2888,7 +2888,7 @@ var _ = Describe("AppManager Tests", func() {
 				Expect(resources.Count()).To(Equal(2))
 
 				rs, ok := resources.Get(
-					serviceKey{"foo", 80, "default"}, "openshift_default_https")
+					serviceKey{"foo", 80, "default"}, "https-ose-vserver")
```

Comment:
Can user specify virtual server names other than "https-ose-vserver" and "ose-vserver" ? The name can be configured when start OpenShift F5 router.  
```
$ oadm router \
    --type=f5-router \
    --external-host-http-vserver=ose-vserver \
    --external-host-https-vserver=https-ose-vserver \
```
see https://docs.openshift.com/container-platform/3.5/install_config/router/f5_router.html

Reply:
Sure, I can add in the ability to override these default names via command line arguments.